server/views/partials/offenceDetailsSection.njk

```diff
@@ -79,6 +95,21 @@
                             }) }}
                         </form>
                     {% endif %}
+
+                    {% if attributes.isEligibleForEncouragementOffence %}
+                        <form method="post" action="/admin/nomis/offence/encouragement">
+                            <input type="hidden" name="_csrf" value="{{ attributes.csrfToken }}" />
+                            <input type="hidden" name="offenceId" value="{{ offence.id }}" />
+                            <input type="hidden" name="offenceCodeSearch" value="{{ attributes.offenceCodeSearch }}" />
+                            {{ govukButton({
+                                classes: "govuk-button--secondary govuk-!-margin-right-2",
+                                text: "Add Enforcement Offence",
```

Comment:
Encouragement rather than Enforcement?

Reply:
Looks like I went snow blind here. I'll update to the correct text.